js/main.js

```diff
@@ -88,12 +116,31 @@ function handleContentLoad(event) {
 }
 
 function newEntryClick(event) {
-  switchView('entry-form');
+  switchView('entry-form', 'new');
 }
 
-function switchView(view) {
+function switchView(view, form = null) {
   for (var i = 0; i < $viewList.length; i++) {
     if ($viewList[i].getAttribute('data-view') === view) $viewList[i].classList.remove('hidden');
     else $viewList[i].classList.add('hidden');
   }
+  var $pageTitle = document.querySelector('.form-page-title');
+  if (form === 'edit') $pageTitle.textContent = 'Edit Entry';
+  else if (form === 'new') $pageTitle.textContent = 'New Entry';
```

Comment:
Instead of needing to add a new conditional to your function to change views, you can change this header text to "Edit Entry" when the user clicks the edit icon, and change it back to "New Entry" every time the form is submitted, so that you can always change it at the appropriate moment, and there will not be a need for an additional conditional